test/mavsdk_tests/process_helper.py

```diff
@@ -238,10 +238,10 @@ def __init__(self,
         super().__init__(log_dir, model, case, verbose)
         self.name = "gzclient"
         self.cwd = workspace_dir
-        self.env = {"PATH": os.environ['PATH'],
+        self.env = dict(os.environ.copy(), **{"PATH": os.environ['PATH'],
```

Comment:
Wow I've never seen something like this before. I think you can remove `PATH`, `HOME`, and `DISPLAY` below, they now all come in automatically.

Reply:
I had that already, just forgot to push again 👍

Reply:
:)